src/components/UserModal/UserModal.tsx

```diff
@@ -131,7 +124,7 @@ const UserModal: React.FC<ModalProps> = observer(
                     <div className={styles.buttonContainer}>
                       <Button
                         className={styles.buttonAuth}
-                        onClick={handleLoginClick}
+                        onClick={() => store.login()}
```

Comment:
Здесь и ниже по коду аналогично, где уместно
```suggestion
                        onClick={store.login}
```